src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
I see what you mean, the problem is from a consumer's view, the wrapper should be obvious what each property is and how its going to return back. Having dynamic passed back up also makes it very fragile for the consumer of the package. There should be some defined set of values as that's the whole point of an API its just a contact on how to communicate. Might be worth putting a model together that captures the bare minimum that you require at the moment and back fill this later on?